bigchaindb/commands/utils.py

```diff
@@ -151,6 +158,10 @@ def mongodb_host(host):
                          help='Specify the location of the configuration file '
                               '(use "-" for stdout)')
 
+base_parser.add_argument('-l', '--log-level',
+                         choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+                         help='Log level')
+
```

Comment:
Sorry, I forgot to write down that I would like to address the documentation part in a separate issue, namely: #1308